src/GeoJson/BoundingBox.php

```diff
@@ -1,4 +1,6 @@
-<?php declare(strict_types=1);
+<?php
+
+declare(strict_types=1);
```

Comment:
Did you manually apply the PSR-12 coding style, or was this the result of some tooling? 

We use `phpcs` and `phpcbf` with the Doctrine coding style on mongo-php-library (see: [phpcs.xml.dist`](https://github.com/mongodb/mongo-php-library/blob/master/phpcs.xml.dist) and [.github/workflows/coding-standards.yml](https://github.com/mongodb/mongo-php-library/blob/master/.github/workflows/coding-standards.yml)), so I would consider adding something similar for this repository. I assume using PSR-12 as-is will have a more concise configuration than that project's customized Doctrine config.

Reply:
If I remember correctly, I temporarily pulled in PHP CS Fixer, ran it, and then removed it again 😅. Being consistent with what you already know totally makes sense - I already had a look at the config in the mongo-php-library and will adapt it to be based on PSR-12 but keep all the custom rules and exclusions except those that aren't needed anymore because of the updated PHP version. 🤞